pkgs/development/python-modules/python-arango/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pyjwt
+, requests
+, requests-toolbelt
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "python-arango";
+  version = "7.5.0";
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1qnvnyrb6djzq2hhr7473cbqj57cxkyyfc9qlv5rgc572fwh3nkd";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    requests-toolbelt
+    pyjwt
+    setuptools
+  ];
+
```

Comment:
Missing `pythonImportsCheck`

Reply:
Thank you! I didn't even realize that existed.